README.md

```
@@ -1,3 +1,57 @@
-# Propels AI
+<div align="center">
+  <img src="https://raw.githubusercontent.com/Propels-AI/Propels/main/apps/extension/public/icon-dark-128.png" width="128" height="128">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)